backend/monitor/monitors.go

```diff
@@ -0,0 +1,170 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/mvcc/mvccpb"
+	"github.com/sensu/sensu-go/backend/store"
+	"github.com/sensu/sensu-go/types"
+)
+
+var (
+	monitorPathPrefix = "monitors"
+	monitorKeyBuilder = store.NewKeyBuilder(monitorPathPrefix)
+)
+
+// Service is the monitors interface.
+type Service interface {
+	// RefreshMonitor starts a new monitor.
```

Comment:
// RefreshMonitor starts a new monitor or resets an existing monitor

Would that be more accurate?